packages/kit/src/runtime/server/page/render.js

```diff
@@ -291,8 +284,8 @@ export async function render_response({
 
 	if (page_config.csr) {
 		const opts = [
+			`app: import(${s(prefixed(client.app.file))})`,
 			`assets: ${s(assets)}`,
```

Comment:
`assets` and `version` should get the same treatment as `env`. that way we can import `app` eagerly, and everything will be a bit simpler

Reply:
ok this was a less straightforward change than i initially imagined. will take a run at it tomorrow